lib/puppet_litmus/serverspec.rb

```diff
@@ -146,17 +152,23 @@ def run_bolt_task(task_name, params = {}, opts = {})
 
     result = run_task(task_name, target_node_name, params, config: config_data, inventory: inventory_hash)
 
-    raise "task failed\n`#{task_name}`\n======\n#{result}" if result.first['status'] != 'success' && opts[:expect_failures] != true
+    result_obj = {
+      exit_code: 0,
+      stdout: (result.first['result']['_output'] unless result.first['result']['_output'].nil?),
```

Comment:
Wasn't sure if this was idiomatic or not, but it was short and didn't require another line of code

Reply:
What does this return if it is nil? Isn't it just nil anyway?

Reply:
You're correct, good catch!